strategy-network-or-cache/README.md

```diff
@@ -0,0 +1,17 @@
+# Network or cache
+The recipe provides a service worker trying to retrieve the most updated content
+but with no detriment of user experience.
+
+## Difficulty
+Beginner
+
+## Use Case
+You want to show the most updated content but not making the user to wait too
+much.
+
+## Solution
+Try serving the content from network but include a timeout to fallback to
```

Comment:
Serve content from the network, but include a timeout to fall back to cached data if the answer from the network doesn't arrive on time.